packages/devtools_app/lib/src/screens/app_size/app_size_screen.dart

```diff
@@ -208,9 +208,7 @@ class _AppSizeBodyState extends State<AppSizeBody>
                   ),
                   Row(
                     children: [
-                      if (currentTab.key == AppSizeScreen.analysisTabKey &&
-                          isDeferredApp)
-                        _buildAppUnitDropdown(),
+                      if (isDeferredApp) _buildAppUnitDropdown(currentTab.key!),
```

Comment:
add some spacing here:

```dart
if (isDeferredApp) ...[
  _buildAppUnitDropdown(currentTab.key!),
  const SizedBox(width: defaultSpacing),
],
```

Reply:
Added spacing so that it only impacts the visuals when both drop downs are there.